lib/mayaUsd/ufe/ProxyShapeHierarchy.cpp

```diff
@@ -179,7 +185,33 @@ Ufe::Group ProxyShapeHierarchy::createGroupCmd(const Ufe::PathComponent& name) c
 {
 	throw std::runtime_error("ProxyShapeHierarchy::createGroupCmd not implemented");
 }
-#endif
+
+#else
+
+Ufe::SceneItem::Ptr ProxyShapeHierarchy::createGroup(const Ufe::Selection& selection, const Ufe::PathComponent& name) const
+{
+	Ufe::SceneItem::Ptr createdItem = nullptr;
```

Comment:
Since these are shared_ptr, you don't really need '= nullptr'

Reply:
true